mne/epochs.py

```diff
@@ -144,7 +144,7 @@ class _BaseEpochs(ProjMixin, ContainsMixin, UpdateChannelsMixin,
     This class provides basic functionality and should never be instantiated
     directly. See Epochs below for an explanation of the parameters.
     """
-    def __init__(self, info, data, events, event_id, tmin, tmax,
+    def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
```

Comment:
Since subclasses, in their own constructor handle `tmax` and `tmin`, modifying to default here isn't necessary, although adds consistency.